docs/setup_eecs281.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: spec
+title: EECS 281 Project Setup
+---
+
+
+EECS 281 Project Setup
+======================
+
+This tutorial walks through EECS 281 project setup using EECS 280 tutorials.
+
+## Visual debugger
+Set up your project in your visual debugger.  We'll use the project name `p1-stats` in this example, but your project name may be different.  If your project has starter files, you'll need the link from the project spec.
+
+| [VS Code Tutorial](https://eecs280staff.github.io/tutorials/setup_vscode.html)| [Visual Studio Tutorial](https://eecs280staff.github.io/tutorials/setup_visualstudio.html) | [Xcode Tutorial](https://eecs280staff.github.io/tutorials/setup_xcode.html) |
+
+<div class="primer-spec-callout warning" markdown="1">
+Some EECS 281 professors are unfamiliar with VS Code.  VS Code works great for EECS 281 projects.  Just ask one of the IAs or GSIs who use VS Code if you need help.
+</div>
+
+After you're done, you should have a folder with a main file.  Your files may be different.
+```console
+$ tree p1-stats/
+p1-stats/
+├── main.cpp
+    ...
+```
+
+## Makefile
+Next, we'll add the [EECS 281 Makefile](https://gitlab.umich.edu/eecs281/makefile).
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** Make sure you're in the directory containing your source code.
+```console
+$ ls
+main.cpp
+```
+</div>
+
+Download the EECS 281 Makefile.
+```console
+$ wget https://gitlab.umich.edu/eecs281/makefile/-/raw/main/Makefile
+$ make help
+EECS281 Advanced Makefile Help
+...
+```
+
+Edit these three lines in the Makefile.  Your values might be different, check the project spec for required file names.
+```make
+UNIQNAME = not_awdeorio
+# ...
+IDENTIFIER  = copy_me_from_project_spec
+# ...
+PROJECTFILE = main.cpp
+# ...
+EXECUTABLE = main
+```
+{: data-title="Makefile" }
+
+**Shortcut:** You can skip updating `PROJECTFILE` if your `main()` function is in one of these filenames: `main.cpp`, `project0.cpp`, `project1.cpp`, `project2.cpp`, `project3.cpp`, `project4.cpp`, `EXECUTABLE.cpp`.
```

Comment:
Add "or" between `project4.cpp` and `EXECUTABLE.cpp` to be grammatically correct.